deepspeed/ops/transformer/inference/op_binding/vector_matmul.py

```diff
@@ -0,0 +1,22 @@
+import torch
+from ..config import DeepSpeedInferenceConfig
+from .base import BaseOp
+
+
+class VectorMatMulOp(BaseOp):
+    def __init__(self, config: DeepSpeedInferenceConfig):
+        super(VectorMatMulOp, self).__init__()
+        self.config = config
+        if self.config.fp16:
+            self.vector_matmul_func = self.inference_cuda_module.vector_matmul_fp16
+        else:
+            self.vector_matmul_func = self.inference_cuda_module.vector_matmul_fp32
+
+    def forward(self,
+                input: torch.Tensor,
+                weight: torch.Tensor,
+                q_scale: torch.Tensor,
```

Comment:
Since `q_scale` is always an attribute of the `weight`, I think we can collapse these two arguments into one.